### DISCUSSION TITLE
Lesson 4: SSLError when trying to deploy to rinkby testnet using infura

### DISCUSSION THREAD

**Discussion by CedricVo91** [smartcontractkit/full-blockchain-solidity-course-py#558] [Q&A]
Dear all, 

Whenever I try to deploy it to the rinkby test net, I get the following error: 

> "C:\Users\cedi4\anaconda3\lib\site-packages\requests\adapters.py", line 514, in send
>     raise SSLError(e, request=request)
> requests.exceptions.SSLError: HTTPSConnectionPool(host='rinkeby.infura.io', port=443): Max retries exceeded with url: /v3/263d25faf7bf4990a6b3360c491558fd (Caused by SSLError("Can't connect to HTTPS URL because the SSL module is not available."))

I double-checked my code and I did everything like in the video. I hope you guys can help me out.

**Comment by cromewar** [comment]
Hello @CedricVo91 as far as I remember another user and the problem was related to the infura/network configuration, try to follow this steps:

1. Make sure you are using the correct endpoints for infura:
![image](https://user-images.githubusercontent.com/20199180/145898240-962ca25d-1879-4c28-a37f-8c102f7c7683.png)
2. If you have the infura key on `.env` file, make sure to export them. (if you are on windows you might have to add them to Path varaibles using windows advanced configuration).
3. As you are using anaconda python, [thread](https://stackoverflow.com/questions/54135206/requests-caused-by-sslerrorcant-connect-to-https-url-because-the-ssl-module) might be helpful for you.

I hope this info might help. Best regards, Cromewar
